### PR TITLE
Fixed translation export path

### PR DESCRIPTION
**Pull request by yasir91**

### CODE REVIEW

File: src/Console/AdminLteInstallCommand.php [hunk 1]
```diff
@@ -274,7 +274,7 @@ protected function exportTranslations()
             }
         }
 
-        CommandHelper::directoryCopy($this->packagePath('resources/lang'), base_path('resources/lang'), $this->option('force'), true);
+        CommandHelper::directoryCopy($this->packagePath('resources/lang'), base_path('resources/lang/vendor/adminlte'), $this->option('force'), true);
```
Comment (mnishihan) [inline src/Console/AdminLteInstallCommand.php]:
```suggestion
        CommandHelper::directoryCopy($this->packagePath('resources/lang'), resource_path('lang/vendor/adminlte'), $this->option('force'), true);
```

**Comment by yasir91** [comment]
what happened?

**Comment by lacodimizer** [comment]
issue on travis

**Comment by yasir91** [comment]
Can I do something for this?

**Comment by lacodimizer** [comment]
I have restarted the build, we will see :)

**Pull request merged**